crates/driver/src/domain/competition/solution/trade.rs

```diff
@@ -83,32 +80,13 @@ impl Fulfillment {
     /// Custom prices are calculated using the uniform clearing prices and the
     /// fee. So that custom prices represent the actual traded amounts as
     /// seen from the user perspective: the amount going in/out of their wallet
-    pub fn calculate_custom_prices(
+    pub fn custom_prices(
```

Comment:
Can we lift the return type into this module (since it's no longer specific to scoring). Also the error type could/should probably just be Math (as it's the only possible error for this computation)

Reply:
fixed both